### PR TITLE
Added Dockerfile and added syft in requirements.txt

### PR DESCRIPTION
**Pull request by kamathhrishi**
Addresses Issue #119.

The Dockerfile can be simply built by running in the root directory

`docker build -t sympc -f docker-images/Dockerfile .`

Further, it can be run by

`docker run -i -t sympc`

### CODE REVIEW

**Review by gmuraru — APPROVED** [review]
LGTM!

**Review by aanurraj — APPROVED** [review]
👍🏻

File: docker-images/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.7-slim-buster
+
+RUN apt-get -y update && apt-get -y upgrade
+RUN apt-get -y install git clang build-essential
+
+COPY ./ SyMPC/
+
+WORKDIR /SyMPC
+
+RUN pip3 install setuptools_scm
```
Comment (gmuraru) [inline docker-images/Dockerfile:10]: Q: Is this required?

Comment (kamathhrishi) [reply docker-images/Dockerfile:10]: Yeah it's required, it was a fix for some bug that was coming.

**Review by gmuraru — APPROVED** [review]
LGTM!

**Pull request merged**